src/rollback_interface.cpp

```diff
@@ -211,6 +210,8 @@ bool RollbackAction::applyRevert(Map *map, InventoryManager *imgr, IGameDef *gam
 					<< inventory_location << std::endl;
 				return false;
 			}
+
+			const std::string &real_name = gamedef->idef()->getAlias(inventory_stack.name);
```

Comment:
exact, i changed the scope and removed the temp variables to use it insinde condition, this prevent a costly getAlias